cypress/e2e/data-import/importing-marc-bib-files/matching-on-newly-created-035-does-not-work.cy.js

```diff
@@ -72,7 +70,26 @@ describe('ui-data-import: Matching on newly-created 035 does not work (regressio
     acceptedType: NewJobProfile.acceptedDataType.marc
   };
 
-  before(() => {
+  before('create test data', () => {
+    cy.getAdminToken().then(() => {
+      InventorySearchAndFilter.getInstancesByIdentifierViaApi(resourceIdentifierForFirstInstance.value)
+        .then(instances => {
+          if (instances) {
+            instances.forEach(({ id }) => {
+              InventoryInstance.deleteInstanceViaApi(id);
+            });
+          }
+        });
+      InventorySearchAndFilter.getInstancesByIdentifierViaApi(resourceIdentifierForSecondInstance.value)
+        .then(instances => {
```

Comment:
rename response maybe like "secondInstancesResponse" or something like that

Reply:
Done